.pydocstyle

```diff
@@ -0,0 +1,3 @@
+[pydocstyle]
+convention = google
```

Comment:
We are using reST docstring for the codebase. Could you change it to that?

Reply:
Thanks for the feedback @refeed. I will make the suggested amendments.